src/plugins/unified_search/public/query_string_input/text_based_languages_editor/esql_documentation_sections.tsx

```diff
@@ -29,30 +39,119 @@ from index | stats average = avg(field) by field2
+              
+\`\`\`
+FROM index
+\`\`\`
+            `,
+              description:
+                'Text is in markdown. Do not translate function names, special characters, or field names like sum(bytes)',
+            }
+          )}
+        />
+      ),
+    },
+    {
+      label: i18n.translate('unifiedSearch.query.textBasedLanguagesEditor.documentation.row', {
+        defaultMessage: 'ROW',
+      }),
+      description: (
+        <Markdown
+          markdown={i18n.translate(
+            'unifiedSearch.query.textBasedLanguagesEditor.documentation.row.markdown',
+            {
+              defaultMessage: `### ROW
+The \`ROW\` source command produces a row with one or more columns with values that you specify. This can be useful for testing.
               
 \`\`\`
-from index
+ROW a = 1, b = "two", c = null
+\`\`\`
+
+ROW supports the use of functions:
+
+\`\`\`
+ROW a = ROUND(1.23, 0)
+\`\`\`
+            `,
+              description:
+                'Text is in markdown. Do not translate function names, special characters, or field names like sum(bytes)',
+            }
+          )}
+        />
+      ),
+    },
+    {
+      label: i18n.translate('unifiedSearch.query.textBasedLanguagesEditor.documentation.show', {
+        defaultMessage: 'SHOW',
+      }),
+      description: (
+        <Markdown
+          markdown={i18n.translate(
+            'unifiedSearch.query.textBasedLanguagesEditor.documentation.show.markdown',
+            {
+              defaultMessage: `### SHOW
+The \`SHOW <item>\` source command returns information about the deployment and its capabilities:
+
+* Use \`SHOW INFO\` to return the deployment's version, build date and hash.
+* Use \`SHOW FUNCTIONS\` to return a list of all supported functions and a synopsis of each function.
+            `,
+              description:
+                'Text is in markdown. Do not translate function names, special characters, or field names like sum(bytes)',
+            }
+          )}
+        />
+      ),
+    },
+  ],
+};
+
+export const processingCommands = {
+  label: i18n.translate('unifiedSearch.query.textBasedLanguagesEditor.processingCommands', {
+    defaultMessage: 'Processing commands',
+  }),
+  description: i18n.translate(
+    'unifiedSearch.query.textBasedLanguagesEditor.processingCommandsDescription',
+    {
+      defaultMessage: `Processing commands change an input table by adding, removing, or changing rows and columns. ESQL supports the following processing commands.`,
+    }
+  ),
+  items: [
+    {
+      label: i18n.translate('unifiedSearch.query.textBasedLanguagesEditor.documentation.dissect', {
+        defaultMessage: 'DISSECT',
+      }),
+      description: (
+        <Markdown
+          markdown={i18n.translate(
+            'unifiedSearch.query.textBasedLanguagesEditor.documentation.from.markdown',
```

Comment:
This key is also used above. Use something like `unifiedSearch.query.textBasedLanguagesEditor.documentation.dissect.markdown`